
<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Writings and Perambulations &#8212; Mutaku  documentation</title>
    
  <link rel="stylesheet" href="_static/css/index.f658d18f9b420779cfdf24aa0a7e2d77.css">

    
  <link rel="stylesheet"
    href="_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      
  <link rel="stylesheet"
    href="_static/vendor/open-sans_all/1.44.1/index.css">
  <link rel="stylesheet"
    href="_static/vendor/lato_latin-ext/1.44.1/index.css">

    
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <link rel="stylesheet" href="_static/basic.css" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/custom.css" />
    <link rel="stylesheet" type="text/css" href="_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="_static/js/index.d3f166471bb80abb5163.js">

    <script id="documentation_options" data-url_root="" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/doctools.js"></script>
    <link rel="author" title="About these documents" href="about/" />
    <link rel="index" title="Index" href="genindex/" />
    <link rel="search" title="Search" href="search/" />
    <link rel="next" title="About Matthew Martz" href="about/" /> 
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="en" /> 
<link
  rel="alternate"
  type="application/atom+xml"
  href="blog/atom.xml"
  title="Mutaku"
/>
 
<style type="text/css">
  ul.ablog-archive {
    list-style: none;
    overflow: auto;
    margin-left: 0px;
  }
  ul.ablog-archive li {
    float: left;
    margin-right: 5px;
    font-size: 80%;
  }
  ul.postlist a {
    font-style: italic;
  }
  ul.postlist-style-disc {
    list-style-type: disc;
  }
  ul.postlist-style-none {
    list-style-type: none;
  }
  ul.postlist-style-circle {
    list-style-type: circle;
  }
</style>

  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    
    <nav class="navbar navbar-light navbar-expand-lg bg-light fixed-top bd-navbar" id="navbar-main">
<div class="container-xl">

    <a class="navbar-brand" href="#">
    
      <img src="_static/plot_header.png" class="logo" alt="logo" />
    
    </a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-menu" aria-controls="navbar-menu" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar-menu" class="col-lg-9 collapse navbar-collapse">
      <ul id="navbar-main-elements" class="navbar-nav mr-auto">
        
        
        <li class="nav-item ">
            <a class="nav-link" href="about/">About Matthew Martz</a>
        </li>
        
        
      </ul>


      

      <ul class="navbar-nav">
        
          <li class="nav-item">
            <a class="nav-link" href="https://github.com/mutaku/" target="_blank" rel="noopener">
              <span><i class="fab fa-github-square"></i></span>
            </a>
          </li>
        
        
          <li class="nav-item">
            <a class="nav-link" href="https://twitter.com/backpropagating" target="_blank" rel="noopener">
              <span><i class="fab fa-twitter-square"></i></span>
            </a>
          </li>
        
      </ul>
    </div>
</div>
    </nav>
    

    <div class="container-xl">
      <div class="row">
          
          <div class="col-12 col-md-3 bd-sidebar"><div class="logo"><img src="_static/profile.jpg"></div>
<div class="snippet">
    <div class="name">Matthew Martz, PhD</div>
    <div class="title">Director of Data Science/ML/AI and Technical Lead / Strategic Vision</div>
    <div class="specialties">Machine learning | Predictive Modeling | Data Science | Biomedical Research | Strategy and Vision | Leadership | Clinical AI | Startups</div>
</div>
          </div>
          

          
          <div class="d-none d-xl-block col-xl-2 bd-toc">
              

<nav id="bd-toc-nav">
    <ul class="nav section-nav flex-column">
    
    </ul>
</nav>


              
          </div>
          

          
          <main class="col-12 col-md-9 col-xl-7 py-md-5 pl-md-5 pr-md-4 bd-content" role="main">
              
              <div>
                 <div class="section" id="writings-and-perambulations">
<h1>Writings and Perambulations<a class="headerlink" href="#writings-and-perambulations" title="Permalink to this headline">¶</a></h1>
<ul class="simple">
<li><p>New Post! Summer of AI - <a class="reference external" href="https://mutaku.io/summer_of_AI/">An AgBiome Perspective (interview)</a></p></li>
</ul>
<p>My expertise is in building and leading teams of Data Scientists, Machine Learning Engineers, Data Engineers, and Data Product experts of diverse experience levels, distributed across cultural and knowledge bases, to drive innovation, product development, and cutting edge, actionable research. I have a proven track record of mentoring, growing, executing, and building strategy around ambitious goals across the Data Science technical platform to provide business value.</p>
<p>My background is in Quantitative Cell Biology and I have a PhD in Biochemistry and Molecular Biology. I spent much of my career researching novel cancer therapeutics through live cell imaging and machine learning. You can now find me developing and implementing machine learning platforms and algorithms for everything from optimization problems to personalized medicine and medicinal AI. I have researched and built novel recommenders, 24/7 production machine learning solutions, and received patents for personalization solutions in the consumer sales space and chemical profile recommenders.</p>
<p>Most recently, I have been working in the Biotechnology space where I have developed the long-term strategy and vision for Data Science, led a growing team of Data Scientists to develop novel algorithms in the predictive modeling and Bioinformatics spaces, and accelerated biological discovery. Moreover, I designed and implemented novel algorithms to detect differential modes of action in both natural products and whole microbe systems, and engineered a proprietary online learning platform to serve as an artificial intelligence decision guidance system for our extensive microbial collection and proprietary product discovery platform (Genesis), a digital twin. I rebuilt global research and enablement programs across domains for knowledge generation and product creation and delivery using cutting edge deep learning techniques on unstructured records, genomics, omics, and image data as the global lead for digital science discovery and delivery programs. Established, built, and implemented data governance and provenance at both the discovery and enterprise levels. Built multimodal knowledge graphs using LLM guidance to develop GraphRAG technologies in-house for production use in the global pipeline.</p>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-body docutils">
<p class="card-text">I am now pursuing my passion work in improving patient outcomes utilizing cutting edge, clinical AI. The company is in early inception phases, but we are already building out the next generation of clinical AI technology. I would love to discuss with you the exciting work we are doing, so please reach out!</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Currently</p>
</div>
<div class="card-body docutils">
<p class="card-text">Mayo Clinic (Rochester, MN) - Multiple roles spanning teams
Additional research focus is on modernizing predictive pipelines and modeling capabilities through the utilization of knowledge graphs, agentic agents, analytical AI, focusing on clinical implementation. I work primarily with Multimodal models and solution from naive builds to fine tuning foundation models and GraphRAG.
My goal is to provide clinicians with predictive assesment data, therapeutic planning strategies, and risk identification and scoring to lower clinician fatigue and improve patient outcomes.
As part of this work, I am currently developing an algorithm and model platform to identify cognitive complexity for each patient touchpoint and the summation to the clinician’s day load.
Please reach out to chat about my research, or any of the work below.</p>
<p class="card-text">Technical Lead and Lead Solution Architect (Transformation Hub) - Rochester, MN</p>
<p class="card-text">Leading technical development of the Transformation Hub products and core infrastructure assets and  partnering with product to realize needs through solutioning and execution</p>
<p class="card-text">Collaborating tightly with product management to create a cohesive execution strategy and ways of work</p>
<p class="card-text">Serving as the technical liaison between the Transformation Hub and other Mayo teams and initiatives</p>
<p class="card-text">Designed and executing the overarching Transformation Hub  technical and product architecture that focuses on future state technologies and capabilities to bring the highest value to CDH and the Mayo Clinic through innovation acceleration, knowledge generation, analytics engines, artificial intelligence, data and model assets, and modular for efficiency and scalability</p>
<p class="card-text">Developing a Knowledge Hub utilizing data scraping and aggregation to build an advanced knowledge graph using an LLM and exposing these data through  APIs, chat, document generation, and product integration through GraphRAG approaches - LLM-created multimodal knowledge graph supporting GraphRAG, Analytical Agents, and direct queries for institutional knowledge understanding, resource planning, and enablement opportunities
Developing an Intake and Triage web application that will utilize the Knowledge Hub and CDH Capabilities models through artificial intelligence to guide innovators and CDH customers/stakeholders to the quickest realization of CDH service - this system aims to further provide CDH with internal insights around gaps, opportunities, staffing, and the ability to build capacity balancing into the general Intake and Triage engine</p>
<p class="card-text">Senior Analytics Architect (Solution Enablement) - Rochester, MN</p>
<p class="card-text">Serving as a team leader for a group of analytics architects to provide solutioning to Center for Digital Health (CDH)  and Enterprise needs</p>
<p class="card-text">Developing an artificial intelligence system to provide self service analytic, dashboard, and insights capabilities from Mayo Clinic data encompassing practice information to capabilities writ large</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Experience</p>
</div>
<div class="card-body docutils">
<p class="card-text">17 years of production Data Science/ML/AI experience and leadership</p>
<p class="card-text">17 years of rigorous scientific training in academic programs - healthcare and precision medicine</p>
<p class="card-text">11+ years solutioning ML/AI architecture</p>
<p class="card-text">8+ years of Data Science and Machine Learning Engineering Leadership and Strategy Development at the Director or VP level</p>
<p class="card-text">5 patents awarded or in final status for the development and application of novel applications of ML/AI</p>
<p class="card-text">25+ Member teams across disciplines in data and product</p>
<p class="card-text">21 years of production Software Engineering experience</p>
<p class="card-text">8 years of ML/AI and advanced analytics product management</p>
<p class="card-text">Industry proven leader in Data Science and Machine Learning Innovation across the AI landscape</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Previously</p>
</div>
<div class="card-body docutils">
<p class="card-text">GLOBAL TRAITS DIGITAL SCIENCE LEAD, GLOBAL TRAITS DISCOVERY AND DELIVERY PROGRAM LEAD</p>
<p class="card-text">BENCHLING RESEARCH PIPELINE TECHNICAL LEAD</p>
<p class="card-text">Led Trait (gene, phenotype) Discovery and Delivery efforts within RD and IT Digital Science across the entire Syngenta global space</p>
<p class="card-text">Created, built, and led Generative AI program to automate and optimize the delivery process for gene delivery processes to target specific traits (phenotypes) or gene expression objectives</p>
<p class="card-text">Created and led the program to use multimodal modeling with structured and instructed data, and foundation LLMs for knowledge generation, product discovery and delivery pipeline performance guidance, regulatory audits, and prompt-based experimental design</p>
<p class="card-text">Owned stack from data model, application landscape, and AI research initiatives from gene discovery to trait (phenotype) introgression including building of API integrations, MLOps, DevOPs workflows</p>
<p class="card-text">Team comprised business (Analysts, Architects, Delivery Managers, Scrum Masters), IT (Developers, QA, MLOps, DevOPs), and science (SMEs, Product Owners, Wet Lab Researchers) domains - both domestic and distributed teams</p>
<p class="card-text">Mentored junior team members with a focus on professional development and upskilling opportunities</p>
<p class="card-text">Led Data Science efforts to research and deploy foundational Large Language Models (LLMs) for protein design to predict expression levels of various molecular biology constructs as a software workbench for bench researchers - work emphasized multimodal data and contextualization/fine tuning of foundation models</p>
<p class="card-text">Built strategy and vision across discovery and delivery to streamline scientific application portfolio, create single source of truth data streams, and initiated foundational work to bring cutting edge ML/AI technologies to the scientific pipeline</p>
<p class="card-text">Communicated strategy and work initiatives to  secure funding and oversee resourcing of execution</p>
<p class="card-text">Collaborated with stakeholders across research and business domains to ensure cooperative acceleration and growth</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Previously</p>
</div>
<div class="card-body docutils">
<p class="card-text">DIRECTOR OF DATA SCIENCE/ML/AI AND TECHNICAL LEAD/STRATEGIC VISION</p>
<p class="card-text">Served as director of Data Science/Machine Learning/Artificial Intelligence strategy and innovation initiatives.</p>
<p class="card-text">Created, built, deployed, then led the genomic AI program for disease prediction, novel mode of action identification, biomarker discovery, and understanding population variation from DNA to signaling pathways/omics.</p>
<p class="card-text">Served as technical lead for the development and utilization of mixed effects models to explain the impact of environmental factors on phenotypic outcomes in the background of genomic models.</p>
<p class="card-text">Led a growing team of Data Scientists working to drive biological research and accelerate product development.</p>
<p class="card-text">Team size ranged from 7 to over 20, across disciplines of Data and Product.</p>
<p class="card-text">Developed and drove 4 year strategy and vision for Data Science, Data Engineering, Bioinformatics, and Data Product for entire organization.</p>
<p class="card-text">Onboarded artificial intelligence methodologies like Generative AI and Large Language Models for genomic information and protein variant creation.</p>
<p class="card-text">Owned stack from data model, application landscape, and AI research initiatives across all of ML/AI and Data Science  including building of API integrations, MLOps, DevOPs workflows</p>
<p class="card-text">Advocate and conduct learning around Data Science/ML/AI products and platforms both to internal teams, executive team and board members, and external partners and collaborators (customers).</p>
<p class="card-text">Directly contributed to high impact publications and invited for speaking engagements.</p>
<p class="card-text">Served as Data Science liaison across the company, communicating strategy, accomplishments, initiatives, and best practices. Developed the long-term Data Science strategy and vision, hiring plan, technological innovation pipeline, and overall project management for the department.</p>
<p class="card-text">Developed a proprietary predictive artificial intelligence decision guidance system to serve as a core to our proprietary natural and whole microbe product discovery platform - GENESIS, a digital twin technology in part combining genomics and geospatial data modeling. This digital twin technology has already generated a large corpus of actionable research, product leads, and accelerated screening paradigms to put the best leads in the field. Excitingly, my team was able to harness GENESIS to generate cross-indication predictive models to drive and accelerate lead identification across the research platform.</p>
<p class="card-text">Led Data Science contribution to external manuscripts and internal white papers for both research, Data Science, and SOPs.</p>
<p class="card-text">Sat on a team of technical leaders that drove research initiatives across the company and served as a hive mind to solve challenges across domains. Identified and addressed gaps in research and Data Science.</p>
<p class="card-text">Developed a novel algorithm to use interpretable machine learning model ensembles to traverse genomic annotations and drive mode of action discovery from small datasets. Designed and developed a predictive modeling platform to accelerate product discovery in indication screening. Designed and developed a cross-indication prediction platform to enable multi-target identification.</p>
<p class="card-text">Two publications and multiple patentable IP technologies developed within first year.</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Previously</p>
</div>
<div class="card-body docutils">
<p class="card-text">DIRECTOR OF DATA SCIENCE/ML/AI AND PRINCIPAL, RESEARCH AND MACHINE LEARNING</p>
<p class="card-text">Was responsible for  envisioning and creating Data Science from scratch that saw a rapid incline of over 10x user growth and key KPIs like LTV, spend, and user ratings through my creation of our AI-driven personalization program.</p>
<p class="card-text">Created and maintained the core, patented algorithms behind the Firstleaf wine club using both shallow, rules-based, and deep learning machine learning and AI technologies on both molecular data and marketing big data and ran the DevOps and MLOps for the realtime, 24/7 AI platform.</p>
<p class="card-text">Designed, built, deployed, then led a set of  interpretable model algorithms to generate industry first user profiles built on billions of data points per user.</p>
<p class="card-text">Designed, built, deployed, then led a data-driven product creation AI built on molecular and consumer profile data, optimized through MCMC parameterizarion, that could scope down to zip code level targets and was used in both standard creation workflows, as well as running regional wine clubs like the LA Times.</p>
<p class="card-text">I built and led a local and distributed team of Data Scientists and Machine Learning engineers (The Research and Machine Learning Team) developing machine learning and AI platforms to drive real time recommendations, inform business strategy, and create/integrate with product design life cycle. This included the end-to-end development of the ML stack using both traditional ML and cutting edge deep learning techniques including computer vision, generative AI, and NLP, along with novel algorithm development.</p>
<p class="card-text">Team size ranged from 5 to 15, across disciplines of Data and Product.</p>
<p class="card-text">I was further responsible for the continual growth of the team, maintaining stakeholder communication, driving Data Science strategy across the organization, and directly working with C-level executives to maintain a vision and communicate strategy and execution plans aligned with business executives.</p>
<p class="card-text">The Research and Machine Learning team was responsible for identifying and developing key Data Science and Machine Learning technologies for Firstleaf. We utilized cutting edge approaches to both empower internal company function as well as customer facing products. The team was also responsible for design and implementation of the patented (developed technologies, co-write and secured patents) algorithms that drove the Firstleaf experience.</p>
<p class="card-text">The team was responsible for initiating, driving, and executing on data science strategies across Marketing, Finance, Business Intelligence, and Wine Making functions – the Research and Machine Learning team was a full spectrum B2B and B2C solution within Firstleaf.</p>
<p class="card-text">5 patents (3 awarded, 2 in final review), multiple interviews, blog posts, and department awards providing high visibility to intellectual property and achievements of team.</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Previously</p>
</div>
<div class="card-body docutils">
<p class="card-text">POSTDOCTORAL RESEARCH FELLOW (Biological Machine Learning and AI) - University of North Carolina, Lineberger Cancer Center</p>
<p class="card-text">American Heart Association funded research fellow in AI-driven precision medicine</p>
<p class="card-text">Identifying, modeling, and understanding noise in single cell signaling during stress responses focusing on utilization of live cell imaging and machine learning on big data</p>
<p class="card-text">Built and ran Data Science and Data Engineering capabilities in the areas of computer vision, natural language processing, artificial intelligence, infrastructure, high performance computing, predictive modeling, and mathematical modeling</p>
<p class="card-text">Built and maintained live cell imaging infrastructure, developed a suite of machine learning algorithms to understand noise in single cell signaling, wrote and secured two fellowships, directly contributed to high impact publications and invited for speaking engagements, and mentored several graduate students and postdoctoral fellows.</p>
<p class="card-text">Extensive experience working with time series data from signal data from engineering of pipelines and early data processing to complex machine learning algorithm development and implementation for decision-making and novel hypothesis generation</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Expertise</p>
</div>
<div class="card-body docutils">
<p class="card-text">Predictive Modeling | Medicinal AI | Machine Learning | Strategy and Vision Building | Resource Allocation Logistics | Algorithm Development | Technical Writing | Leadership | Mentorship | Team Building | Cross-departmental Collaboration | Data Science | Data Engineering | Artificial Intelligence | Generative AI | Python/Software Engineering | Cell Biology | Biophysics | Microscopy | Cancer Therapeutics | E-commerce | Manuscript and Grant Preparation | Patent Development | Bioinformatics | Microbiology | Digital Twins | Biotechnology | Precision Medicine | Multimodal Modeling | Generative AI | Fine-Tuning | AI Trust and Safety | Digital Product Management | AI Validation | Testing and Experimentation | B2B and B2C | Data and Analytics | Data Governance | Regulations and Compliance</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">LANGUAGES, TOOLING, INFRASTRUCTURE</p>
</div>
<div class="card-body docutils">
<p class="card-text">Python | Amazon Web Services (AWS) | Pandas | Numpy | Scikit-learn | Matplotlib | Plotly | Tableau | Scipy | SQL | PyTorch | Sympy | Flask | Gunicorn | FastAPI | BASH | Linux | Git Github | Jupyter | HTML | Javascript | Microsoft Office | Google Suite | Matlab | Full Stack Dev | APIs | Agile | Jira | Time Series Modeling | Simulations | A/B Testing | Multi-armed Bandits | Causal Inference | MLOps | DevOPs | Generative AI| LLMs | NLP (OpenAI, Hugging Face, Vertex) | MultiModal | GenAI | Fine Tuning | Knowledge Graphs | Computer Vision | Snowflake | Databases | DBT | Cloud Computing and Infrastructures (multiple)</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Patents</p>
</div>
<div class="card-body docutils">
<p class="card-text">Systems and methods for labeling and distributing products having multiple versions with recipient version correlation on a per user basis</p>
<p class="card-text">Method, system, and computer readable medium for labeling and distributing products having multiple versions with recipient version correlation on a per user basis</p>
<p class="card-text">Systems and methods for controlling production and distribution of consumable items based on their chemical profiles</p>
<p class="card-text">Using FI-RT to build wine classification models</p>
<p class="card-text">Using FI-RT to generate wine shopping and dining recommendations</p>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Selected Press</p>
</div>
<div class="card-body docutils">
<ul class="simple">
<li><p class="card-text">AI - An AgBiome Perspective (interview)</p></li>
<li><p class="card-text"><a class="reference external" href="https://agbiome.com/genesis-platform/">https://agbiome.com/genesis-platform/</a></p></li>
<li><p class="card-text"><a class="reference external" href="https://www.fastcompany.com/90724383/most-innovative-companies-data-science-2022">https://www.fastcompany.com/90724383/most-innovative-companies-data-science-2022</a></p></li>
<li><p class="card-text"><a class="reference external" href="https://www.modernretail.co/startups/inside-firstleafs-data-driven-approach-to-wine-subscriptions/">https://www.modernretail.co/startups/inside-firstleafs-data-driven-approach-to-wine-subscriptions/</a></p></li>
</ul>
</div>
</div>
</div>
</div>
</div>
<div class="sphinx-bs container pb-4 docutils">
<div class="row docutils">
<div class="d-flex col-lg-12 p-0 docutils">
<div class="card w-100 shadow docutils">
<div class="card-header text-secondary font-weight-bold docutils">
<p class="card-text">Selected Publications</p>
</div>
<div class="card-body docutils">
<p class="card-text">Laura K. Potter, Matthew K. Martz*, Douglas Lawton*.
<a href="#id1"><span class="problematic" id="id2">*</span></a>These authors contributed equally to the work. Ground Truthed Models to Inform Tangible Guids of Global Microbial Diversity Using Deep Neural Network Computer Vision. In Preparation.</p>
<p class="card-text">Yong Jun Goh*, Brody J. DeYoung, Nicholas C. Dove, Brant R. Johnson, Matthew K. Martz, Patrick Videau. AgBiome: Harnessing the Microbial World for Human Benefit. Trends in Biotechnology. 2023.</p>
<p class="card-text">McCarter PC, Vered L, Martz MK, Errede BE, Dohlman, HG, Elston, TC. Temporal separation of opposing MAPK feedback loops leads to robust stress adaptation. In preparation.</p>
<p class="card-text">Ramona Schrage, …, Matthew Martz, …, Evi Kostenis. The experimental power of FR900359 to study Gq-regulated biological processes. Nature Communications 6, Article number: 10156. 14 December 2015.</p>
<p class="card-text">Michelle C Helms, Elda Grabocka, Matthew K Martz, Christopher C Fischer, Nobuchika Suzuki, Philip B Wedegaertner. Mitotic-dependent phosphorylation of leukemia-associated RhoGEF (LARG) by Cdk1. Cellular Signalling, Volume 28, Issue 1, January 2016, Pages 43-52.</p>
<p class="card-text">Martz MK, Grabocka E, Beeharry N, Yen TJ, Wedegaertner PW. Leukemia-Associated RhoGEF (LARG) is a Novel RhoGEF in Cytokinesis and Required for the Proper Completion of Abscission. Mol. Biol. Cell September 15, 2013 vol. 24 no. 18 2785-2794.</p>
<p class="card-text">Matthew Martz and Philip Wedegaertner: Faculty of 1000 Biology, 23 Jul 2010 F1000Prime.com/4242964#eval4039063</p>
<p class="card-text">Carkaci-Salli N, Flanagan JM, Martz MK, Salli U, Walther DJ, Bader M, Vrana KE. Functional domains of human tryptophan hydroxylase 2 (hTPH2). J Biol Chem. 2006 Sep 22;281(38):28105-12. Epub 2006 Jul 24.</p>
</div>
</div>
</div>
</div>
</div>
<p>Here is a list of most recent posts:</p>
<ul class="postlist-style-none postlist">
<li class="ablog-post"><p class="ablog-post-title">09 November - <a class="reference internal" href="summer_of_AI/">Summer of AI - An AgBiome Perspective</a></p>
<p class="ablog-post-excerpt">This is an interview that served as the starting point for a podcast wherein we discussed
Artificial Intelligence with an AgBiome perspective. The podcast went beyond what is below and
looked at the broader societal perspective; I will link the podcast shortly.</p>
</li>
<li class="ablog-post"><p class="ablog-post-title">17 April - <a class="reference internal" href="notes_on_mlops_one/">Notes on MLOps - One</a></p>
<p class="ablog-post-excerpt">This is a short piece I wrote while at Firstleaf as a response
to a really great article on the state of MLOps. I used several
strong points in the article to articulate my thoughts on where
we did things well and directions I would like to see us take.</p>
</li>
<li class="ablog-post"><p class="ablog-post-title">13 November - <a class="reference internal" href="python_generators_and_comprehension/">Python Generators and Comprehension</a></p>
<p class="ablog-post-excerpt">Digging into generators and comprehension - from basics to
to implementation in a comprehensive tutorial. This is a walkthrough
for beginners that will build up to real world examples.</p>
</li>
<li class="ablog-post"><p class="ablog-post-title">13 November - <a class="reference internal" href="nested_dictionary_lookups/">Dictionary Lookup - Exploring the Depths</a></p>
<p class="ablog-post-excerpt">Exploring methods of performant Python dictionary lookups</p>
</li>
</ul>
<div class="toctree-wrapper compound">
</div>
</div>

<div class="section">
   
</div>

              </div>
              
              
              <div class='prev-next-bottom'>
                
    <a class='right-next' id="next-link" href="about/" title="next page">About Matthew Martz</a>

              </div>
              
          </main>
          

      </div>
    </div>

    
  <script src="_static/js/index.d3f166471bb80abb5163.js"></script>


    <footer class="footer mt-5 mt-md-0">
  <div class="container">
    <p>
          &copy; Copyright 2023, Matthew Martz.<br/>
        Created using <a href="http://sphinx-doc.org/">Sphinx</a> 3.5.4.<br/>
    </p>
  </div>
</footer>
  </body>
</html>